assets/src/stories-editor/blocks/amp-story-page/background-color-settings.js

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
Note: moved this part out of the Page block since the file was growing huge. Same for Page Settings. Would be good to perhaps separate Media handling as well but since that's worked on in another PR, left it out.

Reply:
Nice! Long overdue :-)